### ISSUE TITLE
New alerts to help users understand that they need to move down to rub harder

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#124]
Based on recent interviews, we need to cue better how to rub the Chemistry book more productively. We will cue the "down" better and to "rub fast or slow", too.
1. Add explicit "move down" help on the users first to visits two the "At top" edge. So the first 3 visits to the top would be like this:
- First visit:  “At top. Move down to rub harder.” 
- Second visit: “At top. Move down to rub harder.” 
- Third visit: "At top."
- and then begin the count again after a reset.

2. "Down" fires only if the user does not reach the Physics book by the end of the drag. The user actually needs to know they are going down even when they actually hit the book. Hitting the book often happens in a single key press, so "down" is not uttered very often. This is causing some confusion. 
- To fix: In addition to saying "down" at the end of the drag where the users are not actually touching the Physics book, also indicate down when they reach the actual Physics book. On the first 2 times they hit the book, the alert should be:
- Down ((bonk)). Rub fast or slow.
  - So, this means when they hit the book, they hear both the bonk sound and the alert “Down. Rub fast or slow.”
  - Again, do the additional cuing the first 2 times they make contact with the book, and then don’t do  it again until after a reset. Hopefully the user learns that pressing down is a good thing to do.
  - And of course, when they hit down and they do not hit the book, they just get, “Down”.

@zepumph, I edited "At top" string to "At top. Move down to rub harder." to keep to the same sentence structure as the down alert.

**Comment by terracoda** [comment]
@zepumph, let em know if you have any questions about down.
I am going to post a somewhat related issue about left-right rubbing when books are not touching.

**Comment by zepumph** [comment]
This makes total sense to me. No further questions.

**Comment by zepumph** [comment]
We don't yet have repetition when you continue to press against an edge, but other alerts are working. @terracoda please review.

**Comment by zepumph** [comment]
Please see #116 for implementation of the repeated border alerts.

**Comment by terracoda** [comment]
@zepumph, is it possible to publish a dev link for me? It will be easier for me for the next few days.

**Comment by terracoda** [comment]
@zepumph, I tested a built, downloaded version today (October 9th).

The "At top Move down to rub harder." is working great, but is missing a period, after "At top". 
- [ ] It should be, "At top. Move down to rub harder."

The first two "Down. Rub fast or slow."  alerts look good and fire correctly.
However, as I mentioned in https://github.com/phetsims/friction/issues/116#issuecomment-428163257, there is some inconsistency with the plain "Down" alert. I'll move that comment to this issue:

- [ ] I noticed I don't always hear "down", and in some cases I hear "left" or "right" instead of "down". After the first two "Down. Rub fast or slow." alerts are exhausted, the down alerts seem less predictable. I think my https://github.com/phetsims/friction/issues/124#issue-364628801 was not clear and complete. I have rewritten it here and I have added bold highlights where I have added clarifications: 

To fix **the down alerts**: In addition to saying "Down" at the end of the drag where the users are not actually touching the Physics book, also **always** indicate "Down" when they reach the actual Physics book. **In addition**, on the first 2 times they hit the physics book, the alert should be **the more explicit longer alert**:
- Down ((bonk)). Rub fast or slow.
  - So, this means when they hit the book the **first two times**, they hear both the bonk sound and the alert “Down. Rub fast or slow.”
    - Again, do the additional cuing the first 2 times they make contact with the book, and then don’t do it again until after a reset. Hopefully the user learns that pressing down is a good thing to do.
  - **On the third time they hit the Physics book** (and on subsequent times until a reset) they just get a down alert with a simultaneous bonk sound.
    - Down ((bonk)).
  - And of course, when they hit down and they do not hit the book, they just get, “Down”.

**Comment by terracoda** [comment]
Back to you @zepumph.

**Comment by terracoda** [comment]
@zepumph, the At top and Down alerts are sounding good to me. I don't think we need "Down" repeated if user is already against book and presses down. If anything is needed at this point, it would have to cue rubbing left and right. I want to avoid saying, "Against book". 

If the initial down "Down (bonk). Rub fast or slow." isn't enough we can discuss tweaks in a new issue. I'll close this one for now because it is working well.
